pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -454,17 +462,48 @@ stdenv.mkDerivation ({
     runHook postInstall
   '';
 
-  passthru = passthru // {
+  passthru = passthru // rec {
 
     inherit pname version;
 
     compiler = ghc;
 
-
     getBuildInputs = {
-      inherit propagatedBuildInputs otherBuildInputs allPkgconfigDepends;
-      haskellBuildInputs = isHaskellPartition.right;
```

Comment:
I think someone could be relying on this. If so, needs an alias.

Reply:
(or any of these in getBuildInputs)

Reply:
note too that this exists:

https://github.com/NixOS/nixpkgs/blob/acadaa800b566f6237d12428044662e9ec942097/pkgs/development/haskell-modules/lib.nix#L315